actors/builtin/miner/policy.go

```diff
@@ -39,9 +45,8 @@ const SectorsMax = 32 << 20 // PARAM_FINISH
 
 // The maximum number of partitions that may be required to be loaded in a single invocation.
 // This limits the number of simultaneous fault, recovery, or sector-extension declarations.
-// With 48 deadlines (half-hour), 200 partitions per declaration permits loading a full EiB of 32GiB
-// sectors with 1 message per epoch within a single half-hour deadline. A miner can of course submit more messages.
-const AddressedPartitionsMax = 200
+// We set this to same as MaxPartitionsPerDeadline so we can process that many partitions every deadline.
+const AddressedPartitionsMax = 3000
```

Comment:
I wonder if we should just set this to `= MaxPartitionsPerDeadline`.

Reply:
Done.